src/routes/(inner)/components/file-dropzone/+page.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+	import DataTable from '$lib/components/Table/DataTable.svelte';
+	import FileDropzone from '$lib/components/FileDropzone/FileDropzone.svelte';
+	import CodeBlock from '$lib/utilities/CodeBlock/CodeBlock.svelte';
+
+	let files: FileList[] = [];
+
+	function formatter(file: any): any {
```

Comment:
`File`. Do not return `any`, you already know the exact format of the object being returned.